src/layup/orbitfit.py

```diff
@@ -35,6 +35,20 @@
 
 logger = logging.getLogger(__name__)
 
+# The list of required input column names for the provided observations to be fit.
+# Note: This should not include the primary id column name.
+REQUIRED_INPUT_OBSERVATIONS_COLUMN_NAMES = [
+    "ra",
+    "dec",
+    "obstime",
+]
+
+# The list of required column names for the guessed orbits (if provided).
+# Note: This should now include the primary id column name.
+REQUIRED_INPUT_GUESS_COLUMN_NAMES = [
+    "epochMJD_TDB",
+]
```

Comment:
@mschwamb Please let me know if there are columns here that I've missed.

Reply:
We need FORMAT in there as well - ideally the other columns depend on what the input format is - not sure that's doable - otherwise at least checking these two is important allows doing finer checking later

Reply:
Added `FORMAT`. The guess data is run through `convert` which has substantial column checking logic built in. So we should be ok here to just require epochMJD_TDB and FORMAT.